backend/apps/webui/internal/migrations/017_add_user_oauth_sub.py

```diff
@@ -35,7 +35,7 @@ def migrate(migrator: Migrator, database: pw.Database, *, fake=False):
 
     migrator.add_fields(
         "user",
-        oauth_sub=pw.TextField(null=True, unique=True),
+        oauth_sub=pw.CharField(max_length=255, null=True, unique=True),
```

Comment:
Not sure how this change is relevant.

Reply:
MySQL requires the unique field has a key length.

I didn't know the typical `oauth_sub` text size, is there 255 enough? or we should specified the unique key prefix length?

https://stackoverflow.com/questions/1827063/mysql-error-key-specification-without-a-key-length